Jenkinsfile

```
@@ -19,11 +19,20 @@ def mavenEnv(Map params = [:], Closure body) {
       timeout(120) {
         withChecks(name: 'Tests', includeStage: true) {
           infra.withArtifactCachingProxy {
-            withEnv([
-              'JAVA_HOME=/opt/jdk-' + params['jdk'],
-              "MAVEN_ARGS=${env.MAVEN_ARGS != null ? MAVEN_ARGS : ''} -B -ntp -Dmaven.repo.local=${WORKSPACE_TMP}/m2repo"
+            cache(
+                // max cache size in MB, will be reset after exceeding this size
+                maxCacheSize: 2048
```

Running a clean build to see how big this is, close to 1gb currently

---

```suggestion
                maxCacheSize: 20480
```

=> let's go for 20 Gb

- Controller available size:

```bash
$ df -h /var/lib/jenkins
Filesystem      Size  Used Avail Use% Mounted on
/dev/sdb        503G  201G  277G  43% /var/lib/jenkins
```

- Currently used data in ACP replicas (data not compressed on disk):

```
$ kubectl -n artifact-caching-proxy exec -ti artifact-caching-proxy-0 -- df -h /data/nginx-cache
Filesystem                Size      Used Available Use% Mounted on
/dev/sdb                 97.9G     17.2G     80.6G  18% /data/nginx-cache

$ kubectl -n artifact-caching-proxy exec -ti artifact-caching-proxy-1 -- df -h /data/nginx-cache
Filesystem                Size      Used Available Use% Mounted on
/dev/sdb                 97.9G     17.8G     80.0G  18% /data/nginx-cache
```